src/Satooshi/Bundle/CoverallsV1Bundle/Config/Configuration.php

```diff
@@ -191,7 +191,7 @@ public function addCloverXmlPath($cloverXmlPath)
     /**
      * Return absolute path to clover.xml.
      *
-     * @return string
+     * @return array
```

Comment:
try to update https://github.com/satooshi/php-coveralls/blob/master/devtools/composer.json#L7
to `friendsofphp/php-cs-fixer: ^1.12`, maybe it would help